### ISSUE TITLE
Windfree and fan direction control

### ISSUE THREAD

**Issue by snappytoes1** [0x4a616e/homebridge-smartthings-ac#12]
It will be great if you add support of the Windfree settings and control of the air flow direction

**Comment by 0x4a616e** [comment]
Yes, i fully agree. The only caveat is that this is (at least last time I checked) not supported by HomeKit for heater/cooler devices... That means, that some sort of additional HomeKit device would be required to allow for that kind of capability... And things start to get a bit messy... I'd really love to see this but at this point, i don't really have a good solution for this :/

**Comment by rgreco87** [comment]
+1 for this feature!

**Comment by jeazyee** [comment]
How about adding a virtual power-strip for each ac unit. The powerstrip for the unit has then switches for windfree, silent and turbo.
